cg/cli/workflow/nf_analysis.py

```diff
@@ -193,3 +193,34 @@ def store_housekeeper(context: CGConfig, case_id: str, dry_run: bool) -> None:
     except HousekeeperStoreError as error:
         LOG.error(f"Could not store bundle in Housekeeper and StatusDB: {error}!")
         raise click.Abort()
+
+
+@click.command("store")
+@ARGUMENT_CASE_ID
+@OPTION_DRY
+@click.pass_context
+def store(context: click.Context, case_id: str, dry_run: bool) -> None:
+    """Generate deliverable files for a case and store in Housekeeper if they
+    pass QC metrics checks."""
+    analysis_api: NfAnalysisAPI = context.obj.meta_apis[MetaApis.ANALYSIS_API]
+
+    is_latest_analysis_qc: bool = analysis_api.trailblazer_api.is_latest_analysis_qc(
+        case_id=case_id
+    )
+    if not is_latest_analysis_qc and not analysis_api.trailblazer_api.is_latest_analysis_completed(
+        case_id=case_id
+    ):
+        LOG.error(
+            "Case not stored. Trailblazer status must be either QC or COMPLETE to be able to store"
+        )
+        raise click.Abort
+
+    if (
+        is_latest_analysis_qc
+        or not analysis_api.get_metrics_deliverables_path(case_id=case_id).exists()
+    ):
+        LOG.info(f"Generating metrics file and performing QC checks for {case_id}")
+        context.invoke(metrics_deliver, case_id=case_id, dry_run=dry_run)
+    LOG.info(f"Storing analysis for {case_id}")
+    context.invoke(report_deliver, case_id=case_id, dry_run=dry_run)
+    context.invoke(store_housekeeper, case_id=case_id, dry_run=dry_run)
```

Comment:
```suggestion
    analysis.api.report_deliver()
    analysis_api.store_housekeeper()
```

Following the `invoke`it looks like the analysis_api  that is initiated here has these functions that are invoked via the CLI. Why do we not call them directly here?

Reply:
Seems to also be the case for the `metrics_deliver`